Sources/Plot/API/ControlFlow.swift

```diff
@@ -26,8 +26,11 @@ public extension Node {
     /// be created.
     /// - parameter optional: The optional value to unwrap.
     /// - parameter transform: The closure to use to transform the value into a node.
-    static func unwrap<T>(_ optional: T?, _ transform: (T) throws -> Node) rethrows -> Node {
-        try optional.map(transform) ?? .empty
+    /// - parameter defaultValue: The closure to use when the value is nil.
+    static func unwrap<T>(_ optional: T?,
+                          _ transform: (T) throws -> Node,
+                          _ defaultValue: () -> Node = { .empty }) rethrows -> Node {
```

Comment:
I think this should follow the same convention as `if` does, in that it should accept an optional `Node`, rather than a closure, and use the `else` parameter label. That way we keep the two APIs consistent. There's also no real need for the "else" clause to be executed as a closure - it just adds syntactic overhead. There *is* a benefit in that it gets lazily evaluated, but I think the difference is negligible in this case, so I'd rather opt for consistency and simplicity in terms of syntax.

Reply:
Compare the following two call sites:

```swift
// Your current implementation:
Node.unwrap(optional, Node.text, { .text("Is nil") })

// My suggested API:
Node.unwrap(optional, Node.text, .text("Is nil"))
```